.cicd/entrypoint.sh

```diff
@@ -8,5 +8,6 @@ mkdir build
 execute cd /workdir/build
 execute ccache -s
 execute cmake -DCMAKE_TOOLCHAIN_FILE=/workdir/.cicd/clang.make -DENABLE_TESTS=ON ..
-execute make -j $(nproc)
+echo Building with -j$JOBS
```

Comment:
This line is unnecessary, as `execute()` prints the number of cores being used, seen [here](https://buildkite.com/EOSIO/eos-vm-beta/builds/214#1d82ecd6-090d-4e07-bf3c-9bdf8c08201e/260). Plus, with code folding on, this print statement will be hidden anyways.

Reply:
Resolved here:
https://github.com/EOSIO/eos-vm/pull/51/commits/207665f45c260ad3f52f33f75d4d2e894e29eac2